src/status_im/contexts/chat/messenger/messages/drawers/view.cljs

```diff
@@ -93,6 +93,11 @@
       (rf/dispatch [:pin-message/send-pin-message
                     (assoc message-data :pinned message-not-pinned?)]))))
 
+(defn message-pin-shown?
+  [community?]
+  (let [able-to-send-message? (rf/sub [:chats/able-to-send-message?])]
```

Comment:
I'm not sure (can't remember) if the rule controlling who can pin is only tied to the idea of _being able to send a message_, but maybe this is all fine @jo-mut, but maybe worth double-checking if you are not sure as well.